src/components/POLineCard/POLineCard.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useAcqMethods } from '@folio/orders/src/common/hooks/useAcqMethods/useAcqMethods';
```

Comment:
If this import can be replaced by [this](https://github.com/folio-org/stripes-acq-components/blob/master/lib/hooks/useAcqMethodsOptions/useAcqMethodsOptions.js) then that should happen. We don't want to rely on @folio/orders